pyproject.toml

```
@@ -42,3 +42,6 @@ build-backend = "poetry.core.masonry.api"
 pythonpath = "bagitobjecttransfer"
 python_files = "test_*.py"
 DJANGO_SETTINGS_MODULE = "bagitobjecttransfer.settings.test"
+
+[tool.poetry.plugins."pytest11"]
```

The `conftest` runs fine on my machine without the quotes around `pytest11`. Can you double check whether they're required on your machine too?

```suggestion
[tool.poetry.plugins.pytest11]
```

---

Addressed by:
https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/344/commits/c682701d7aaee98bd437da98554a8b09ccc381a7